applications/RomApplication/python_scripts/calculate_rom_basis_output_process.py

```diff
@@ -109,6 +113,27 @@ def PrintOutput(self):
                     self.next_output += self.snapshots_interval
 
     def ExecuteFinalize(self):
+        # Initialize the Python dictionary with the default settings
+        # Note that this order is kept if Python 3.6 onwards is used
```

Comment:
Huhhhh this is dangerous 
Are you sure abt this? Do you have a reference?

in the CoSimApp we use OrderedDict for this purpose

Reply:
Well, more than dangerous in this case it would be annoying (if the order is not kept you'll need to browse through the file to manually edit something as the basis arrays might be way long).

In any case, [here](https://stackoverflow.com/questions/1867861/how-to-keep-keys-values-in-same-order-as-declared) it is explained.

Reply:
Cool, I didnt know!